survey/docker.py

```diff
@@ -17,29 +24,89 @@ def _path(self):
 
     def _iidfile(self):
         return self._path() / "image"
-    
+
     def _iid(self):
         return self._iidfile().read_text()
-    
+
     def build(self):
-        subprocess.run(("docker", "build", ".", "--iidfile", self._iidfile(), "--progress", "plain"), cwd=self._path(), check=True)
+        version = subprocess.run(
+            ("git", "describe", "--tags", "--always"),
```

Comment:
Ah is this why fetching tags is necessary?

Reply:
Yes. I need the tags and the commits connecting the checked out commit to the tags in order to generate a version string.